content/pages/tools/jupyter-book.html

```diff
@@ -0,0 +1,137 @@
+{#
+  View this page at http://localhost:8080/tools/jupyter-book/
+#}
+<html>
+  <head>
+    <title>Publish your book on the internet with Read the Docs</title>
+    <meta name="template" content="page" />
+    <meta name="slug" content="tools/jupyter-book" />
+    <meta name="status" content="hidden" />
+    <meta name="description" content="Create an HTML version of your book and publish it online." />
+  </head>
+  <body>
+
+{% import "partials/basic.html" as basic %}
+{% import "partials/marketing.html" as marketing %}
+{% import "partials/technical.html" as technical %}
+{% import "partials/homepage.html" as homepage %}
+
+<section>
+
+  {% call basic.header_image(
+    header="Publish your book on the internet",
```

Comment:
It might be worth copying the `<title>` here, mentioning Jupyter Book in some way

Reply:
```suggestion
    header="Publish your documentation with Jupyter Book and Read the Docs",
```